examples/taskmanager/task_manager_rolling.py

```diff
@@ -0,0 +1,110 @@
+import qlib
+from qlib.config import REG_CN
+from qlib.workflow.task.gen import RollingGen, task_generator
```

Comment:
These functions are cluttered into one script.
Can we create a framework to accommodate them?